html-css/building-blocks/step3-instructions.md

```diff
@@ -24,12 +22,14 @@ Click the <strong>Copy to Editor</strong> button below to add html for a card co
 &lt;/div&gt;
 </pre>
 
-2) <strong>Add the bullseye class to the card so that the card is centered horizontally and vertically on the page.</strong>
+2) <strong>Add the bullseye class to the card.</strong>
+
+This will allow the card to be centered horizontally and vertically on the page.
 
-Find the outer `<div>` wrapper on Line 1 and add the class `pf-l-bullseye`.
+Find the outermost `<div>` wrapper for the card and add the class `pf-l-bullseye`.
 
-<strong>Hint: </strong>`<div class="pf-l-bullseye">`
+It should look like: `<div class="pf-l-bullseye">`
 
-Once the browser reloads, you should see that the card is now centered in the middle of the page!
+Once the preview reloads, the card should be centered in the middle of the page.
 
-<strong>Note: </strong>It’s important to follow the documentation for layouts because it will demonstrate where to add the layout class. The documentation for bullseye is [here](https://www.patternfly.org/v4/documentation/core/layouts/bullseye). In the documentation it specifies that you should add the class `pf-l-bullseye` to the parent container of its child.
+<strong>Note: </strong>It’s important to follow the documentation for layouts because it will demonstrate where to add the layout class. The documentation for bullseye is [here](https://www.patternfly.org/v4/documentation/core/layouts/bullseye). In the documentation it specifies to add the class `pf-l-bullseye` to the parent container of its child.
```

Comment:
```suggestion
<strong>Note: </strong>It’s important to follow the documentation for layouts because it demonstrates where to add the layout class. The [documentation for bullseye] (https://www.patternfly.org/v4/documentation/core/layouts/bullseye) specifies to add the class `pf-l-bullseye` to the parent container of its child.
``` 

We should avoid "click here" terminology. It's included on the "Shadowman Would Never Say" list on Mojo, marked as an archaic term from the old internet. Instead, we should just hyperlink the "documentation for bullseye."